examples/preprocessors__flow-browserify/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "cypress-example-flow-browserify",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "start": "echo",
+    "cypress:run": "../../node_modules/.bin/cypress run",
+    "cypress:open": "../../node_modules/.bin/cypress open"
+  },
+  "devDependencies": {
```

Comment:
These dependencies should be moved to the root package.json https://github.com/cypress-io/cypress-example-recipes/blob/develop/package.json

Running any example project should only require `npm i` from the root, not installing within each individual package.

Reply:
Makes sense. Fixed.